content/id/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md

```diff
@@ -0,0 +1,379 @@
+daripada menggunakan berkas sertifikat yang terpisah. Dalam kasus ini, kamu perlu menambahkan akhiran `-data` ke kunci. Contoh, `certificate-authority-data`, `client-certificate-data`, dan `client-key-data`.
+
+Setiap konteks memiliki tiga bagian, klaster, pengguna, dan Namespace.
+Sebagai contoh, konteks `dev-frontend` menyatakan, "Gunakan kredensial dari pengguna `developer`
+untuk mengakses Namespace `frontend` di klaster `development`.
+
+Mengatur konteks yang digunakan:
+
+```shell
+kubectl config --kubeconfig=config-demo use-context dev-frontend
+```
+
+Sekarang kapanpun kamu memasukkan perintah `kubectl`, aksi tersebut akan diterapkan untuk klaster,
+dan Namespace yang terdaftar pada konteks `dev-frontend`. Dan perintah tersebut akan menggunakan
+kredensial dari pengguna yang terdaftar pada konteks `dev-frontend`.
+
+Untuk melihat hanya informasi konfigurasi yang berkaitan dengan konteks saat ini,
+gunakan `--minify` flag.
+
+```shell
+kubectl config --kubeconfig=config-demo view --minify
+```
+
+Output menunjukkan informasi konfigurasi yang berkaitan dengan konteks `dev-frontend`:
+
+```shell
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority: fake-ca-file
+    server: https://1.2.3.4
+  name: development
+contexts:
+- context:
+    cluster: development
+    namespace: frontend
+    user: developer
+  name: dev-frontend
+current-context: dev-frontend
+kind: Config
+preferences: {}
+users:
+- name: developer
+  user:
+    client-certificate: fake-cert-file
+    client-key: fake-key-file
+```
+
+Sekarang apabila kamu ingin bekerja sebentar di klaster eksperimen.
+
+Ubah konteks saat ini menjadi `exp-scratch`:
+
+```shell
+kubectl config --kubeconfig=config-demo use-context exp-scratch
+```
+
+Sekarang, setiap perintah `kubectl` yang diberikan akan berlaku untuk Namespace `default`
+dari klaster `scratch`. Dan perintah akan menggunakan kredensial dari pengguna yang
+terdaftar di konteks `exp-scratch`.
+
+Untuk melihat konfigurasi yang berkaitan dengan konteks saat ini, `exp-scratch`.
+
+```shell
+kubectl config --kubeconfig=config-demo view --minify
+```
+
+Akhirnya, misalkan kamu ingin bekerja sebentar di Namespace `storage` pada
+klaster `development`.
+
+Ubah konteks saat ini menjadi `dev-storage`:
+
+```shell
+kubectl config --kubeconfig=config-demo use-context dev-storage
+```
+
+Untuk melihat konfigurasi yang berkaitan dengan konteks baru saat ini, `dev-storage`.
+
+```shell
+kubectl config --kubeconfig=config-demo view --minify
+```
+
+## Membuat sebuah file konfigurasi kedua
+
+Di direktori `config-exercise` kamu, buat sebuah berkas bernama `config-demo-2` dengan konten ini:
+
+```shell
+apiVersion: v1
+kind: Config
+preferences: {}
+
+contexts:
+- context:
+    cluster: development
+    namespace: ramp
+    user: developer
+  name: dev-ramp-up
+```
+
+Berkas konfigurasi diatas mendefinisikan sebuah konteks baru bernama `dev-ramp-up`.
+
+## Mengatur variabel lingkungan KUBECONFIG
```

Comment:
let's discuss if we should use "variabel lingkungan" or "variabel _environment_"
cc @kubernetes/sig-docs-id-reviews

Reply:
In RSNI glosarium, `environment` is translated to `lingkungan`

Reply:
make sense 👍